release notes/v1.0.0-rc1.md

```diff
@@ -0,0 +1,222 @@
+k6 `v1.0.0-rc1` is here 🎉!
+
+This release marks a special, non-conventional milestone in the k6 software life-cycle, serving as a preview of the upcoming version 1.0.0.
+
+The purpose of this release is to give the community a chance to test the new version, identify any potential issues, and test migrations of any parts affected by breaking changes. If you encounter any problems or have trouble with the migration, we encourage you to report them by creating an issue. Your feedback will help improve the final release. If no critical issues are reported, we plan to release the final `v1.0.0` within the next month.
+
+Here’s a glimpse of what’s new in this release:
+
+- `k6/experimental/webcrypto` promoted to stable and available globally under `crypto`.
+- A revamped end-of-test summary aiming to bring an easier way to understand test results.
+- `k6/browser` provided an API for tracking network requests and responses.
+
+## Breaking changes
+
+- [#4541](https://github.com/grafana/k6/pull/4541) Commas(`,`) are now supported in the values of the `--tag` CLI flag. This is a breaking change, as previously, a comma meant the start of a new set of tag-values. As a comma is a valid symbol for the value of a tag, this is necessary to have equivalency between different ways of setting tags. This still allows multiple tags to be set on the CLI with multiple `--tag key=value` arguments.
+
+### A revamped end-of-test summary aiming to bring an easier way to understand test results [#4089](https://github.com/grafana/k6/pull/4089)
+
+**TL;DR** This change is a breaking change because the data structure that [`handleSummary`](https://grafana.com/docs/k6/latest/results-output/end-of-test/custom-summary/) function receives,
+has changed. So, existing custom summary handlers need to be adjusted accordingly to the new data structure. 
+Meanwhile, `--summary-mode=legacy` can be used for backwards compatibility. 
+
+Other than that, we offer no guarantees on how the summary looks like, and it might change on new releases again.
+
+```
+ █ TOTAL RESULTS 
```

Comment:
```suggestion
  █ THRESHOLDS

    http_req_duration
    ✓ 'p(95)<1500' p(95)=316.31ms
    ✓ 'p(90)<2000' p(90)=297.46ms

    http_req_failed
    ✓ 'rate<0.3' rate=25.00%
 
   █ TOTAL RESULTS 
```

Reply:
cc @joanlopez as I wrote the docs, I found the thresholds change is actually a big one, and it might be worth showing it here too 🙇🏻

Reply:
@oleiade @joanlopez this seems very big text for release notes, maybe we can move to a collapsed section? We can have a text like `Expand to see a real example of the new End-of-Test summary`. What do you think?

Reply:
I've added a commit that replaces it with an image inside a collapsible section. I stored the image in a dedicated `images` folder, let me know if you'd prefer it done some other way 🙇🏻

Reply:
I prefer this method which doesn't require us to have images in the source code that everybody pools 

https://youtu.be/2ajWUp8F694?si=uYjbe5NtgLCkwZXc&t=48